### PR TITLE
Add service card

### PR DESCRIPTION
**Pull request by andienn**
What's this PR do ? ⭐️
Make service card component

What are the relevant ticket ? ⭐️
https://gteam.atlassian.net/browse/MAIB-66?atlOrigin=eyJpIjoiZjEyY2FmYjE1ZTI4NDg0OWI4ZjQ5NWRiZmM5MGY1NGIiLCJwIjoiaiJ9

Any deployment note (migrations, new envs, rake tasks, cron) ?
Definition of Done:
- [] Does this PR introduces a breaking change ?

- [ ] Does this PR passes Lint locally ?

- [ ] Does this PR requires Database changes ?

- [x] Does this PR requires a regression test by QA ?

- [ ] Does this PR adds new dependencies ? (network setup, new 3rd party services)

Screenshots (if appropriate)
image

### CODE REVIEW

**Review by rezzaldy — APPROVED** [review]

**Comment by rizkybaihaqy** [comment]
conflict @andienn

File: components/ServiceCard/ServiceCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.serviceCard{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 32px 0px;
+    // gap: 40px;
```
Comment (rizkybaihaqy) [inline components/ServiceCard/ServiceCard.module.scss]: delete aja kalo udh ga dipake

File: components/ServiceCard/ServiceCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    // position: relative;
+    width: 298px;
+    height: 363px;
+    left: 16px;
+    top: 16px;
```
Comment (rizkybaihaqy) [inline components/ServiceCard/ServiceCard.module.scss]: top left ini kyknya ga dipake kalo ga absolute

File: components/ServiceCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import styles from "./ServiceCard.module.scss"
+import cn from "classnames";
+import Image from "next/image";
+
+type AppProps = {
```
Comment (rizkybaihaqy) [inline components/ServiceCard/index.tsx:5]: coba pakai cardtype dien

File: components/ServiceCard/ServiceCard.module.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+        .content{
+            color: var(--white);
+        }
+    }
+
+    .media{
+        width: 104px;
+        height: 104px;
+        // top: 32px;
```
Comment (rizkybaihaqy) [inline components/ServiceCard/ServiceCard.module.scss]: kalau udh ga dipake mending di hapus aja dien, jangan cuman dikomment

File: components/ServiceCard/ServiceCard.module.scss [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        position: relative;
+    }
+
+    .title{
+        width: 266px;
+        height: 32px;
+        font-weight: 700;
```
Comment (rizkybaihaqy) [inline components/ServiceCard/ServiceCard.module.scss]: ini lu lupa pake variable font yang udh ada atau emang belum ada variable font ini dien?

File: components/ServiceCard/index.tsx (hunk 3 above)
Comment (andienn) [reply components/ServiceCard/index.tsx:5]: gada date jd gapake cardtype

**Review by rizkybaihaqy — APPROVED** [review]

**Pull request merged**